packages/monitor-v2/src/monitor-og/README.md

```diff
@@ -11,11 +11,21 @@ node ./packages/monitor-v2/dist/monitor-og/index.js
 
 All the configuration should be provided with following environment variables:
 
-- `OG_ADDRESS` is the address of the deployed Optimistic Governor that this bot will monitor.
 - `OG_MASTER_COPY_ADDRESSES` is an array of addresses of the deployed Optimistic Governor master copies used to discover
   new deployments of Optimistic Governor proxies (defaults to protocol deployments)
 - `MODULE_PROXY_FACTORY_ADDRESSES` is an array of addresses of the deployed Module Proxy Factory used to discover new
```

Comment:
there are values here that I think will always be the same/always known. do we not always know the master copy, for example, or these proxy factory addresses? we should have defaulting for these kinds of things to simplify the configs for users and let people override if needed. in general we should aim to have minimal viable confits that require as little setup as possible in most cases.

Reply:
These are optional parameters. By default the bot pulls the mastercopy from networks file and latest factory address from zodiac repo. Users need to set OG_MASTER_COPY_ADDRESSES and MODULE_PROXY_FACTORY_ADDRESSES only if there has been upgrade to either of these and need to capture deployments that were made before upgrade

Reply:
I updated the readme to be more explicit on this here: dbfda2f8f6b84743abb8c219ac9b60eb25856363